cmd/libs/go2idl/client-gen/test_apis/testgroup/v1/types.generated.go

```diff
@@ -159,7 +159,13 @@ func (x *TestType) CodecEncodeSelf(e *codec1978.Encoder) {
 				z.EncSendContainerState(codecSelfer_containerArrayElem1234)
 				if yyq2[2] {
 					yy10 := &x.ObjectMeta
-					yy10.CodecEncodeSelf(e)
```

Comment:
I don't see a way to detect promoted methods from embedded fields via reflection.